### ISSUE TITLE
Time not implemented on this platform on yew-oauth2-example

### ISSUE THREAD

**Issue by dmatos2012** [ctron/yew-oauth2#22]
Hi,
First of all thanks for your amazing work here. :) . 

I am learning bit about `oauth2` and wanted to integrate this on my app, but running the `yew-oauth2-example` as per the readme, seems to log in successfully, but looking at the logs, I get an error regarding `time is not implemented`. Is it expected?

```
docker-compose -f develop/docker-compose.yaml up
cd yew-oauth2-example
trunk serve
```
And then get this:
```
panicked at 'time not implemented on this platform', library/std/src/sys/wasm/../unsupported/time.rs:31:9

Stack:

__wbg_get_imports/imports.wbg.__wbg_new_abda76e883ba8a5f@http://localhost:8080/yew-oauth2-example-967c702dc098d627.js:399:21
console_error_panic_hook::Error::new::he7fcaa1455222268@http://localhost:8080/yew-oauth2-example-967c702dc098d627_bg.wasm:wasm-function[16485]:0x797bef
console_error_panic_hook::hook_impl::hcb1643e4297f9336@http://localhost:8080/yew-oauth2-example-967c702dc098d627_bg.wasm:wasm-function[4020]:0x53c00b
console_error_panic_hook::hook::h14fd4bfad2bc6b1b@http://localhost:8080/yew-oauth2-example-967c702dc098d627_bg.wasm:wasm-function[22790]:0x7fe678
core::ops::function::Fn::call::he16fc9ba676c4956@http://localhost:8080/yew-oauth2-example-967c702dc098d627_bg.wasm:wasm-function[19593]:0x7d150a
std::panicking::rust_panic_with_hook::h9fe353d01bf1e5b0@http://localhost:8080/yew-oauth2-example-967c702dc098d627_bg.wasm:wasm-function[9018]:0x6a4101
```

And this is how the page looks, with empty `<ul>` elements:
![image](https://github.com/ctron/yew-oauth2/assets/11776554/344e9ea4-0b3f-4de7-92c8-e624b8cb12a4)

Thank you :)

**Comment by ctron** [comment]
No this is not expected, but it also rings a bell … I need to google to recall …

**Comment by ctron** [comment]
Ok, this can be fixed by enabling the `wasm-bindgen` feature on the `time` crate:

```toml
time = { version = "0.3", features = ["wasm-bindgen"] }
```

I will update the example and try to enable this by default.

**Comment by ctron** [comment]
It's released as `0.9.2`
